### PR TITLE
Service Registration Sample Implementation

### PR DESCRIPTION
**Pull request by almirvuk**
[[Update] 02-ServiceRegistration #49](https://github.com/PrismLibrary/Prism-Samples-Forms/issues/49)

I have implemented this Service Registration sample using two standard examples.

Page(s):

- ViewAPage
- ViewBPage

Services:

- Interface `IExampleAlphaService `with implementation in `ExampleAlphaService`
- Interface `IExampleBetaService `with implementation in `ExampleBetaService`

Let me know if there is a need to change and edit something.

_P.S. I did not change readme.md file at all, this sample is working 100% just want to see if this example is ok with others as well, after that, I will change `readme.md` file as well. 😄_

### CODE REVIEW

File: 02-ServiceRegistration/src/PrismSample/ViewModels/MainPageViewModel.cs [hunk 1]
```diff
@@ -1,11 +1,33 @@
...
+
+        private async void GoToViewA()
+        {
+            await _navigationService.NavigateAsync(nameof(ViewAPage));
+        }
+
+        private async void GoToViewB()
+        {
+            await _navigationService.NavigateAsync(nameof(ViewBPage));
+        }
```
Comment (dansiegel) [inline 02-ServiceRegistration/src/PrismSample/ViewModels/MainPageViewModel.cs]: use a magic string here... we want to keep this simple and `nameof(SomePage)` is a break of the MVVM pattern

File: 02-ServiceRegistration/src/PrismSample/Views/ViewAPage.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace PrismSample.Views
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
```
Comment (dansiegel) [inline 02-ServiceRegistration/src/PrismSample/Views/ViewAPage.xaml.cs]: remove this we already have it at the assembly level... be sure to remove unused namespaces as well ;)

File: 02-ServiceRegistration/src/PrismSample/Views/ViewBPage.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class ViewBPage : ContentPage
+    {
+        public ViewBPage()
+        {
+            InitializeComponent();
+        }
+    }
+}
```
Comment (dansiegel) [inline 02-ServiceRegistration/src/PrismSample/Views/ViewBPage.xaml.cs]: see comment on ViewA

Comment (almirvuk) [reply 02-ServiceRegistration/src/PrismSample/Views/ViewBPage.xaml.cs]: Done with the latest commit :)

File: 02-ServiceRegistration/src/PrismSample/Views/ViewAPage.xaml.cs (hunk 2 above)
Comment (almirvuk) [reply 02-ServiceRegistration/src/PrismSample/Views/ViewAPage.xaml.cs]: Done with the latest commit :)

File: 02-ServiceRegistration/src/PrismSample/ViewModels/MainPageViewModel.cs (hunk 1 above)
Comment (almirvuk) [reply 02-ServiceRegistration/src/PrismSample/ViewModels/MainPageViewModel.cs]: Done with the latest commit :)

**Comment by dansiegel** [comment]
Looks great. Thanks for the PR

**Pull request merged**